fireant/queries/builder/dataset_blender_query_builder.py

```diff
@@ -184,11 +205,24 @@ def _blender_join_criteria(
 
 
 def _blend_query(dimensions, metrics, orders, field_maps, queries):
-    base_query, *join_queries = queries
-    base_field_map, *join_field_maps = field_maps
+    for i, query in enumerate(queries):
+        if query:
+            queries = queries[i:]
+            field_maps = field_maps[i:]
+            break
+    else:
+        return None
```

Comment:
Looks like the indentation got shifted

Reply:
No that's intended. If the break never gets hit in the for loop, then the else statement is executed.

Reply:
Ah okay I see!